notification-manager/src/main/java/com/qpay/notificationmanager/config/MailMessageGenerator.java

```diff
@@ -0,0 +1,23 @@
+package com.qpay.notificationmanager.config;
+
+import com.qpay.libs.models.PaymentNotification;
+import lombok.RequiredArgsConstructor;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class MailMessageGenerator {
+
+    private final EmailTemplatesProperties emailTemplate;
+
+    public SimpleMailMessage getPaymentMessage(final PaymentNotification paymentNotification) {
+        final var message = new SimpleMailMessage();
+        final var text = String.format(emailTemplate.getPayment().getText(), paymentNotification.amount());
+        message.setTo(paymentNotification.emailTo());
+        message.setSubject(emailTemplate.getPayment().getSubject());
+        message.setText(text);
+        return message;
```

Comment:
where is email from?

Reply:
it's automatically substitutes the mail from application.yamal as the sender